### PR TITLE
configure phpstan for sylius

### PR DESCRIPTION
**Pull request by JanTvrdik**
image

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Thanks for improvement.

Could you add similar change for Spyker as well so all projects are in balance?

File: phpstan-sylius.neon [hunk 1]
```diff
@@ -1,4 +1,12 @@
+php:
+	memory_limit: 2G
+
 parameters:
-    bootstrap: "phpstan-sylius-bootstrap.php"
-    ignoreErrors:
-        - "#Class [A-Za-z\\\\]+ not found and could not be autoloaded#"
+	bootstrap: "project/sylius/vendor/autoload.php"
```
Comment (TomasVotruba) [inline phpstan-sylius.neon]: There should be spaces, as in the rest of project

**Comment by JanTvrdik** [comment]
> Could you add similar change for Spyker as well so all projects are in balance?

Yes I can. But I don't plan to do it.

**Comment by TomasVotruba** [comment]
Without it this PR would create more imbalanced results, which is not an option.

**Comment by JanTvrdik** [comment]
I have no interest in having correct results in your article. I'm only interested in teaching people how to use phpstan. If I fix the other projects for you, you will never learn. I've given you an example, the rest is up to you.

**Comment by TomasVotruba** [comment]
I see. I don't plan to spend more time on this repository at the moment, so I'm only open to PR that will help to improve it, not add extra work.

Thanks for your points, I might get to them when next version of this post would be released, e.g. when Shopsys would be open-sourced and I get more Spryker feedback as well.

**Pull request closed without merging**